content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -100,19 +100,19 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 **Steps**
 
->NOTE: Ensure that the csm-operator is installed and that the namespace, secrets, and config.yaml are created as prerequisites.
+>NOTE: Ensure that the csm-operator is installed and that the namespace, secrets, and `config.yaml` are created as prerequisites.
 
-1. Copy the downloaded values.yaml file.
+1. Copy the downloaded `values.yaml` file.
 
 2. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-3. If Observability is enabled, refer to [Observability](../csmoperator/modules/observability) for post installation dependencies.
+>NOTE: The CSM Installation Wizard generates `values.yaml` with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, you can follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [PowerScale](../../csidriver/installation/helm/isilon/#install-the-driver), [PowerFlex](../../csidriver/installation/helm/powerflex/#install-the-driver), [Unity XT](../../csidriver/installation/helm/unity/#install-csi-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/).
 
-4. If Authorization is enabled, refer to [Authorization](../csmoperator/modules/authorization) for installation and configuration of the Proxy Server.
+3. If Observability is checked in the wizard, only the sidecar is enabled. Refer to [Observability](../../observability/deployment/#post-installation-dependencies) to export metrics to Prometheus and load the Grafana dashboards.
 
->NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
+4. If Authorization is checked in the wizard, only the sidecar is enabled. Refer to [Authorization](../../authorization/deployment/helm/) to install and configure the CSM Authorization Proxy Server.
 
-5. If Replication is enabled, refer to [Replication](../csmoperator/modules/replication) for the necessary prerequisites required for this module.
+5. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
```

Comment:
Same as above. Module gets installed too. Please check the behavior of other modules.
I also see the change in reference link. It should point to the module info from CSM Operator like before.